css/src/components/breadcrumbs.scss

```diff
@@ -11,10 +17,21 @@ $breadcrumb-separator: '\00200B/' !default;
 		padding-block: $breadcrumb-spacing-vertical;
 
 		&:not(.breadcrumbs-item-slashless)::after {
-			display: inline-block;
+			@extend %breadcrumbs-slash;
+
 			margin-inline: $breadcrumb-spacing-horizontal;
-			color: $text-subtle;
-			content: $breadcrumb-separator;
 		}
 	}
+
+	// stylelint-disable selector-max-specificity
+
+	&.breadcrumbs-initial-slash {
+		.breadcrumbs-item:first-child:not(.breadcrumbs-items-slashless)::before,
+		.breadcrumbs-item[hidden] + .breadcrumbs-item:not(.breadcrumbs-items-slashless)::before {
```

Comment:
feel free to resolve: do we need `:not(.breadcrumbs-items-slashless)` in these selectors? I was thinking `.breadcrumbs-items-slashless` was to disable the default trailing slash.